README.md

```diff
@@ -47,7 +47,7 @@ Please follow [the instruction](https://docs.databricks.com/en/dev-tools/cli/dat
 
 To create a new project, run:
 
-    databricks bundle init https://github.com/databricks/mlops-stack
+    databricks bundle init mlops-stacks
```

Comment:
This is awesome. Shall we cut a release and update the alias pointing to the release version later?

Reply:
Yup! We'll be cutting the MLOps Stacks v0.2 release after this PR merges, and deco team has plans to add this semantic versioning to the bundle aliases.